internal/command/run.go

```diff
@@ -88,6 +96,10 @@ type runCmd struct {
 
 	uploadRoutinePool     *routines.Pool
 	taskRunnerRoutinePool *routines.Pool
+	taskRunner            *baur.TaskRunner
+
+	terminateGracefullyOnce sync.Once
```

Comment:
I think it's fine here, `runCmd` is never copied and if it would, the linter would complain (because at least of the mutex in sync.Once that would be copied).